### PR TITLE
Add platform_view_swift based on platform_view example

### PR DESCRIPTION
**Pull request by mblaughton**
This is a port of the Objective-C platform_view example in Swift. I tried to keep it true to the original, and the port matches the Objective-C version's behavior on my device.

### CODE REVIEW

File: examples/platform_view_swift/ios/Runner/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// Copyright 2017, the Flutter project authors.  Please see the AUTHORS file
```
Comment (RedBrogdon) [inline examples/platform_view_swift/ios/Runner/AppDelegate.swift]: I'd bump this to 2018 here and everywhere else.

File: examples/platform_view_swift/lib/main.dart [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                  mainAxisAlignment: MainAxisAlignment.center,
+                  children: <Widget>[
+                    new Text(
+                      'Button tapped $_counter time${ _counter == 1 ? '' : 's' }.',
+                      style: const TextStyle(fontSize: 17.0),
+                    ),
+                    new Padding(
+                      padding: const EdgeInsets.all(18.0),
+                      child: new RaisedButton(
+                          child: Platform.isIOS
```
Comment (RedBrogdon) [inline examples/platform_view_swift/lib/main.dart]: Unless you're planning to add Android (which I wouldn't recommend), I'd nip out this platform check.

**Review by redbrogdon — CHANGES_REQUESTED** [review]
Ian's got a good point. We could land this here and then move it over along with the others, but that's adding useless code to the commit history of a crowded repo.

@mblaughton, how about dropping the entire platform_view_swift directory into the root of flutter/samples? I can come along afterward and update the README and travis script to accommodate it.

**Comment by mblaughton** [comment]
@RedBrogdon wrote: how about dropping the entire platform_view_swift directory into the root of flutter/samples?

Absolutely, will do -- Thanks for the feedback.

I'm away from my development environment for a couple weeks but will make the suggested changes around the week of August 6.

**Comment by RedBrogdon** [comment]
Awesome. 

Thanks for putting this together. I'll go ahead and close this PR, and look for one in `flutter/samples` when you're ready.

**Pull request closed without merging**